### DISCUSSION TITLE
User registration with different user profile

### DISCUSSION THREAD

**Discussion by nexagensoftware** [giuliopons/adadmin#96] [General]
Why publisher when a client registers on their own? Can't choose whether it's an advertiser or not. I want the publisher to have direct access to the codes. the advertiser can record the direct ad.

**Comment by giuliopons** [comment]
Hi,
a Publisher can't register freely because, in my view, the Administrator must accept the Publishers.
So, the creation of a Publisher should be a process quite different from Advertisers registration.
I could add this part if you have a small budget for it, I can make this in the next week for 100 Eur.
Let me know!

**Comment by nexagensoftware** [comment]
The whole program costs $37. only this plugin is 100 euro?
:)

On Fri, Aug 18, 2023 at 4:20 PM Giulio Pons ***@***.***>
wrote:

> Hi,
> a Publisher can't register freely because, in my view, the Administrator
> must accept the Publishers.
> So, the creation of a Publisher should be a process quite different from
> Advertisers registration.
> I could add this part if you have a small budget for it, I can make this
> in the next week for 100 Eur.
> Let me know!
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/giuliopons/adadmin/discussions/96#discussioncomment-6761542>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A7W5LWQHXQGJU7VVGIRJKYDXV5T2JANCNFSM6AAAAAA3F4QAEM>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 


ONLYCPM ADVERTISE- QUALITY TRAFFIC, QUALITY PUBLISHING
***@***.*** - ***@***.***
skype: erdem.kuran   -    -    https://www.onlycpm.online/index.html

**Reply by giuliopons** [reply]
Yes.
I'm sorry for the price, obviously you can't compare this work to the price of the whole software, because this is a new-feature request and I will make it in just a week.
I can lower it a bit, or another user could join this bounty with you.

**Comment by nexagensoftware** [comment]
Why dont open page?
Moodad.com

18 Ağu 2023 Cum 22:58 tarihinde Giulio Pons ***@***.***> şunu
yazdı:

**Reply by giuliopons** [reply]
What do you mean? Please explain the problem clearly.
I can login on your site with credentials you previously sent me by email.

**Comment by nexagensoftware** [comment]
Hi gian how are you .

Advertisers can already enter. But I want a page that publishers can access
alone. Streamers will be able to log in, receive codes instantly and start
making money. How do we do this and how much will it cost me?

On Thu, Aug 24, 2023 at 6:42 PM Giulio Pons ***@***.***>
wrote:

> What do you mean? Please explain the problem clearly.
> I can login on your site with credentials you previously sent me by email.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/giuliopons/adadmin/discussions/96#discussioncomment-6814771>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A7W5LWSB4ZM3A7WQQRTHHSLXW5Y5DANCNFSM6AAAAAA3F4QAEM>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 


ONLYCPM ADVERTISE- QUALITY TRAFFIC, QUALITY PUBLISHING
***@***.*** - ***@***.***
skype: erdem.kuran   -    -    https://www.onlycpm.online/index.html

**Comment by nexagensoftware** [comment]
I bought Adadmin. I'm trying it on my own website, but the banner doesn't
appear? From where?

Publishers say the code does not show ads. I can't answer them, you need to
get back to me urgently. Because you come back too late. I'm planning to
raise it more, one by one if you help me.

On Wed, Nov 15, 2023 at 6:07 PM ONLYCPM ADVERTISE ***@***.***>
wrote:

> Hi gian how are you .
>
> Advertisers can already enter. But I want a page that publishers can
> access alone. Streamers will be able to log in, receive codes instantly and
> start making money. How do we do this and how much will it cost me?
>
> On Thu, Aug 24, 2023 at 6:42 PM Giulio Pons ***@***.***>
> wrote:
>
>> What do you mean? Please explain the problem clearly.
>> I can login on your site with credentials you previously sent me by email.
>>
>> —
>> Reply to this email directly, view it on GitHub
>> <https://github.com/giuliopons/adadmin/discussions/96#discussioncomment-6814771>,
>> or unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/A7W5LWSB4ZM3A7WQQRTHHSLXW5Y5DANCNFSM6AAAAAA3F4QAEM>
>> .
>> You are receiving this because you authored the thread.Message ID:
>> ***@***.***>
>>
>
>
> --
>
>
> ONLYCPM ADVERTISE- QUALITY TRAFFIC, QUALITY PUBLISHING
> ***@***.*** - ***@***.***
> skype: erdem.kuran   -    -    https://www.onlycpm.online/index.html
>
>

-- 




MOODAD ONLYCPM ADVERTISE- QUALITY TRAFFIC, QUALITY PUBLISHING
***@***.*** - ***@***.***
Telegram: ***@***.*** * / ***@***.****

**Reply by giuliopons** [reply]
Mmm your question is too generic.
Have you installed the software?
Have you configured one "position" and placed the "position script" in your site?
Have you uploaded at least one banner in that position?

You can read something about how it works at this address:
https://www.barattalo.it/amb/src/componenti/docs/

**Reply by nexagensoftware** [reply]
I sent the login information to your e-mail address.  Can you check and help me?

**Comment by nexagensoftware** [comment]
Hi
I think I need to update the software first, but how will publishers and
advertisers stay inside?

On Mon, Feb 5, 2024 at 5:59 PM Giulio Pons ***@***.***> wrote:

> Mmm your question is too generic.
> Have you installed the software?
> Have you configured one "position" and placed the "position script" in
> your site?
> Have you uploaded at least one banner in that position?
>
> You can read something about how it works at this address:
> https://www.barattalo.it/amb/src/componenti/docs/
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/giuliopons/adadmin/discussions/96#discussioncomment-8371308>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A7W5LWRDKA7DOXTPEY4ETCTYSDXV7AVCNFSM6AAAAAA3F4QAEOVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4DGNZRGMYDQ>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 




MOODAD ONLYCPM ADVERTISE- QUALITY TRAFFIC, QUALITY PUBLISHING
***@***.*** - ***@***.***
Telegram: ***@***.*** * / ***@***.****

**Reply by giuliopons** [reply]
Please read this article:
https://www.barattalo.it/tools/adadmin-usage-scenarios/

**Reply by nexagensoftware** [reply]
I sent the login information to your e-mail address.  Can you check and help me?

**Reply by nexagensoftware** [reply]
Giulio, you asked me for my panel information. I sent. Could you please look at the problem? Thank you

**Reply by giuliopons** [reply]
> Giulio, you asked me for my panel information. I sent. Could you please look at the problem? Thank you 

Hi,
Yes. I've received your information. I need some time, please be patient. :-)
Giulio

**Comment by nexagensoftware** [comment]
Thank you for your answer.

On Wed, Feb 7, 2024 at 10:23 AM Giulio Pons ***@***.***>
wrote:

> Giulio, you asked me for my panel information. I sent. Could you please
> look at the problem? Thank you
>
> Hi,
> Yes. I've received your information. I need some time, please be patient.
> :-)
> Giulio
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/giuliopons/adadmin/discussions/96#discussioncomment-8391806>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A7W5LWRFOAWDK3SOYEHCSEDYSMTY5AVCNFSM6AAAAAA3F4QAEOVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4DGOJRHAYDM>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>


-- 




MOODAD ONLYCPM ADVERTISE- QUALITY TRAFFIC, QUALITY PUBLISHING
***@***.*** - ***@***.***
Telegram: ***@***.*** * / ***@***.****
